src/graphnet/models/task/task.py

```diff
@@ -64,6 +64,7 @@ def __init__(
         transform_target: Optional[Callable] = None,
         transform_inference: Optional[Callable] = None,
         transform_support: Optional[tuple] = None,
+        loss_weight_column: Optional[str] = None,
```

Comment:
I would suggest that we don't use "column" in the naming. As I see it, "column" is something in a database, whereas here it is an attribute of a graph (a `Data` object).
```suggestion
        loss_weight: Optional[str] = None,
```